src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -15,17 +15,14 @@
             <GroupBox x:Uid="AnalysisGroupBox"
                       Header="{x:Static local:AdvancedOptionPageStrings.Option_Analysis}">
                 <StackPanel>
-                    <Label Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_analysis_scope}"/>
-                    <StackPanel Margin="15, 0, 0, 0">
-                        <RadioButton  GroupName="Background_analysis_scope"
-                                              x:Name="Background_analysis_scope_active_file"
-                                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Active_File}"/>
-                        <RadioButton  GroupName="Background_analysis_scope"
-                                              x:Name="Background_analysis_scope_open_files"
-                                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Open_Files_And_Projects}"/>
-                        <RadioButton  GroupName="Background_analysis_scope"
-                                              x:Name="Background_analysis_scope_full_solution"
-                                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Full_Solution}"/>
+                    <Label Content="{x:Static local:AdvancedOptionPageStrings.Option_Run_background_code_analysis_for}"/>
+                    <StackPanel>
+                        <ComboBox x:Name="Run_background_code_analysis_for" IsEditable="false" AutomationProperties.Name="{x:Static local:AdvancedOptionPageStrings.Option_Run_background_code_analysis_for}">
+                            <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_None}" Tag="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_None_Tag}" />
+                            <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Active_File}" Tag="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Active_File_Tag}" />
+                            <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Open_Files}" Tag="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Open_Files_Tag}" />
+                            <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Full_Solution}" Tag="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Full_Solution_Tag}" />
```

Comment:
hrmm... why does the tag have to reference the 'XXXStrings' type, but it's referring to enum values.. why not just reference the enum directly ?  i don't get it.

Reply:
I took the existing approach used by ColorScheme enum/tag: https://github.com/dotnet/roslyn/blob/7b6b57276d7a27198956beb3a86d7d0db34a0829/src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageStrings.cs#L276-L280